src/main/java/org/candlepin/model/EntitlementCurator.java

```diff
@@ -660,21 +660,26 @@ private int countMatchesByFilters(AbstractHibernateObject<?> object, String obje
     }
 
     /**
-     * List all entitled product IDs from entitlements which overlap the given date range.
+     * Returns the set of all entitled product IDs from consumer's entitlements, current pool &
+     * set of all entitled pools (selected via auto attach or bulk pool attach) which overlap
```

Comment:
Instead of `entitled pools`, let's make this `pools about to be entitled`, to be in line with the new variable name

Reply:
Done.